apps/docs/content/docs/guide/cli.mdx

```diff
@@ -30,11 +30,16 @@ To install the `CLI` globally, execute one of the following commands in your ter
 
 ### Without Installation
 
-Alternatively, you can use the `CLI` without a global installation by employing `npx`:
+Alternatively, you can use the `CLI` without a global installation by running the following commands:
 
-```bash
-npx nextui-cli@latest
-```
+<PackageManagers
+  commands={{
+    npm: "npx nextui-cli@latest",
+    yarn: "yarn dlx nextui-cli@latest",
+    pnpm: "pnpm dlx nextui-cli@latest",
+    bun: "bunx nextui-cli@latest",
```

Comment:
`bunx` does not work. Instead we need to use: `bun x`

Reply:
Maybe it does not work on my end 🙃

![bunx](https://github.com/user-attachments/assets/cde9b6e6-23b3-48a1-8966-eed5254a41e6)

`bun x` always works

![bun x](https://github.com/user-attachments/assets/dea8f682-aae1-46aa-85d7-3859e5a7a5f3)

Reply:
seems a known issue for bun on Windows. I saw there is a issue reported in bun repo.